pkgs/development/python-modules/lockfile/default.nix

```diff
@@ -14,25 +14,28 @@ buildPythonPackage rec {
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "6aed02de03cba24efabcd600b30540140634fc06cfa603822d508d5361e9f799";
+    hash = "sha256-au0C3gPLok76vNYAswVAFAY0/AbPpgOCLVCNU2Hp95k=";
   };
 
+  # The __init__ method doesn't do anything relevant as far as I can tell
+  # So we disable it, since it also makes pytest fail.
+  postPatch = ''
+    substituteInPlace test/compliancetest.py \
+      --replace-fail "def setup" "def setUp" \
+      --replace-fail "def teardown" "def tearDown" \
```

Comment:
I think it would be best to use `setup_method` and `teardown_method` here. Since `substituteInPlace` won’t be able to handle the `saved_class` stuff, maybe you could use a patch file here?

Reply:
Patch file would definitely be better, this is just a quick fix solution. Was planning on that anyways, thanks for the reminder to do so.